mod-pubsub-server/src/main/java/org/folio/services/publish/PublishingServiceImpl.java

```diff
@@ -42,8 +42,8 @@ public PublishingServiceImpl(@Autowired Vertx vertx,
   }
 
   @Override
-  public Future<Boolean> sendEvent(Event event, String tenantId) {
-    Promise<Boolean> promise = Promise.promise();
+  public Future<Void> sendEvent(Event event, String tenantId) {
+    Promise<Void> promise = Promise.promise();
     PubSubConfig config = new PubSubConfig(kafkaConfig.getEnvId(), tenantId, event.getEventType());
     executor.executeBlocking(future -> {
```

Comment:
The `Promise<Void> promise = Promise.promise();` is no longer needed.

```
    return executor.executeBlocking(future -> {
      ...
    });
```

Reply:
done